net/unbound/files/README.md

```diff
@@ -0,0 +1,81 @@
+# Unbound Recursive DNS Server with UCI
+
+## Description
+This package of UNBOUND is prepared with UCI. It allows UNBOUND to be used in parallel or serial to DNSMASQ. Recurse the universe DNS, but forward to DNSMASQ for your DHCP resolved names.
```

Comment:
Okay. Could have just said style issue; the program names probably don't need caps.